app/controllers/users_controller.rb

```diff
@@ -53,7 +53,7 @@ def choose_identity
   
   def update_identity
     new_identity = h(params[:identity].to_s)
-    return redirect_to choose_identity_path unless %w(Student Volunteer Mentor).include?(new_identity)
+    return redirect_to choose_identity_path unless %w(Participant Volunteer Mentor).include?(new_identity)
```

Comment:
`Student` is modeled as a non-participating student, currently used primarily for event registrations where anyone could RSVP even if they weren't actually participants in the program. Students are given essentially zero access to anything, except for event-based actions, like choosing if they're a high-school student or an external volunteer during event RSVP. 

Given that we do event registrations a bit differently now, we might not even need this `update_identity` step.